examples/runtime/starlark/using-path/README.md

```diff
@@ -0,0 +1,21 @@
+# Starlark Path Example
+
+In this example, starlark source is provided using path in the function
```

Comment:
General note: I suggest expanding such sentences and being one level more precise and specific.